sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -910,10 +918,12 @@ the `T` template parameter as follows: +
 `tf32`: `sycl::ext::oneapi::experimental::matrix::precision::tf32` +
 `fp32`: `float` +
 `fp64`: `double` +
+`sint4`: `sycl::ext::oneapi::experimental::matrix::precision::sint4_t` +
```

Comment:
Do `max_msize`, `msize`, etc. tell the number of matrix elements or the number of storage elements when the type is `sint4` or `uint4`?